tutorials/animation/playing_videos.rst

```diff
@@ -116,6 +116,18 @@ See :ref:`doc_viewports` and the
 `GUI in 3D demo <https://github.com/godotengine/godot-demo-projects/tree/master/viewport/gui_in_3d>`__
 for more information on setting this up.
 
+Looping a video
+^^^^^^^^^^^^^^^
+
+For looping a video, the **Loop** property can be enabled. This will seamlessly
+restart the video when it reaches its end.
+
+Note that the project setting **Video Delay Compensation** to a non-zero value
```

Comment:
```suggestion
Note that setting the project setting **Video Delay Compensation** to a non-zero value
```

Slight typo, otherwise looks good to me :)

Reply:
Thank you! I force-pushed the correction 🙂